dspace-api/src/main/java/org/dspace/importer/external/liveimportclient/service/LiveImportClient.java~main

```diff
@@ -0,0 +1,21 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.importer.external.liveimportclient.service;
+
+import java.util.Map;
+
+/**
+ * Interface for classes that allow to contact LiveImport clients.
+ *
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.com)
+ */
+public interface LiveImportClient {
```

Comment:
This class strangely has an extension of `*.java-main`?  Maybe some sort of merge/rebase issue?